File: 151004/Gonzarevich/lab2/DTO/CommentResponseTo.php
<?php

namespace App\DTO;

use App\Entity\Comment;

class CommentResponseTo
{
    public readonly int $id;
    public readonly int $issueId;
    public readonly string $content;

    public function __construct(
        Comment $issue
    ) {
        $this->id = $issue->getId();
        $this->issueId = $issue->getIssueId()->getId();
        $this->content = $issue->getContent();
    }
}

File: 151001/Timoshek/publisher/src/main/java/by/bsuir/egor/dto/CommentResponseTo.java
package by.bsuir.egor.dto;

public class CommentResponseTo {
    private long id;
    private long issueId;
    private String content;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public long getIssueId() {
        return issueId;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public void setIssueId(long issueId) {
        this.issueId = issueId;
    }
}
